### PR TITLE
Inter cpi leveldb 3node

### PR DESCRIPTION
**Pull request by thezhe**
Hashed log syncing with 64bit hashes is DONE!!! Please use the sync function in GossipNode.h. It will sync the hashed logs, request for hash definitions, and then finally sync the unhashed logs.

One issue I had was Common.h did not include leveldb/db.h. I also commented out any leveldb operations in GossipNode while testing. I trust Fuyao knows about this better than I do.

### CODE REVIEW

File: src/CPISync/GossipNode.cc [hunk 1]
```diff
@@ -1,55 +1,102 @@
...
 
+void GossipNode::sync(string host, int NUM_CHAR, bool server)
+{
+    string res;
+    //sync hashes
+    cout << "New hashed logs recieved:" << endl <<endl;
+    if (server)
+    {
+        res = exec("./sync " + host+ " s " + to_string(NUM_CHAR) + getNewHashedLogs());
+        cout<<res <<endl <<endl;
```
Comment (zhou-1) [inline src/CPISync/GossipNode.cc:44]: Why we need 2 endl here?

**Review by zhou-1 — COMMENTED** [review]
Awesome work!

File: src/CPISync/GossipNode.cc (hunk 1 above)
Comment (thezhe) [reply src/CPISync/GossipNode.cc:44]: just to make it look nicer in the terminal

**Comment by aktrinh17** [comment]
Looks great :)

**Review by wfystx — APPROVED** [review]
LGTM
Well done 🆒

**Pull request merged**